docs/resiliency/index.asciidoc

```diff
@@ -134,12 +123,36 @@ We have increased our test coverage to include scenarios tested by Jepsen. We ma
 This status page is a start, but we can do a better job of explicitly documenting the processes at work in Elasticsearch, and what happens in the case of each type of failure. The plan is to have a test case that validates each behavior under simulated conditions. Every test will document the expected results, the associated test code and an explicit PASS or FAIL status for each simulated case.
 
 [float]
-=== Do not allow stale shards to automatically be promoted to primary (STATUS: ONGOING)
+=== Make index creation more user friendly (STATUS: ONGOING)
+
+Today, Elasticsearch returns as soon as a create-index request has been processed,
+but before the shards are allocated.  Users should wait for a `green` cluster health
+before continuing, but we can make this easier for users by waiting for a quorum
+of shards to be allocated before returning.  See {GIT}9126[#9126]
+
+[float]
+=== Do not allow stale shards to automatically be promoted to primary (STATUS: ONGOING, v3.0.0)
+
+In some scenarios, after the loss of all valid copies, a stale replica shard can be assigned as a primary ({GIT}14671[#14671]). This can lead to
+a loss of acknowledged writes if the valid copies are not lost but are rather temporarily isolated. Allocation IDs ({GIT}14739[#14739]) solve this
+issue by tracking non-stale shard copies in the cluster and using this tracking information to allocation primary shards.
```

Comment:
Add: "When all shard copies are lost, Elasticsearch will wait for one of these copies to appear. You can still issue a manual command to reuse stale copies, if present".